_includes/nav.html

```diff
@@ -3,18 +3,37 @@
 - items are pulled from _config.yml
 {% endcomment %}
 
-<header class="nav">
-  <a class="nav__link" href="{{ '/' | relative_url }}">{{ site.title }}</a>
-  <nav class="">
+<header class="nav" data-module="nav" data-mobile="false">
+  <a class="nav__logo" href="{{ '/' | relative_url }}">
+    {% include components/svg.html
+      filename = "logo"
+      class = "nav__svg"
+    %}
+    <span class="nav__title -short">{{ site.short_title }}</span>
+    <span class="nav__title -long">{{ site.title }}</span>
+  </a>
+  <button class="nav__toggle" data-nav-toggle>
+    <div class="nav__toggle__line -top"></div>
+    <div class="nav__toggle__line -middle"></div>
+    <div class="nav__toggle__line -middle -reversed"></div>
+    <div class="nav__toggle__line -bottom"></div>
+  </button>
```

Comment:
Just a heads up that an "empty button" accessibility error is being generated here. One possibility for a fix is to add an `aria-label` attribute.

Reply:
Good callout! I missed this! 🙇